_includes/get-started-band.html

```diff
@@ -2,7 +2,7 @@
   <div class="gs-item">
     <div class="number"><div>1</div></div>
     <div class="text">
-      You need <a href="https://en.wikipedia.org/wiki/Comparison_of_integrated_development_environments#Java" target="_blank">an IDE</a> like IntelliJ IDEA, Eclipse, VSCode or even Vim!
+        You need <a href="https://en.wikipedia.org/wiki/Comparison_of_integrated_development_environments#Java" target="_blank">an IDE</a> like <a href=""www.jetbrains.com/idea/download/">IntelliJ IDEA</a>, <a href="https://www.eclipse.org/downloads/">Eclipse</a>, <a href="https://code.visualstudio.com/Download">VSCode</a> or even <a href="https://spacevim.org/use-vim-as-a-java-ide/">Vim</a> or <a href="http://spacemacs.org">Emacs</a>!
```

Comment:
Do we really really want to have Emacs ;-)

Reply:
It’s the Best of them all - even have a decent text editor ;)